django-stubs/db/models/fields/related_descriptors.pyi

```diff
@@ -60,6 +63,9 @@ class ReverseOneToOneDescriptor(Generic[_From, _To]):
     def get_prefetch_queryset(
         self, instances: list[_From], queryset: QuerySet[_To] | None = ...
     ) -> tuple[QuerySet[_To], Callable[..., Any], Callable[..., Any], bool, str, bool]: ...
+    def get_prefetch_querysets(
```

Comment:
Why these two signatures are different? `ForwardManyToOneDescriptor(Generic[_F])` looks generic too

Reply:
I simply missed it. I thought that the existing code was written that way for some reason. If you could provide an answer above, I will go ahead and modify all the relevant parts of the class accordingly.